.env.example

```diff
@@ -31,43 +24,15 @@ REDIS_HOST=127.0.0.1
 REDIS_PASSWORD=null
 REDIS_PORT=6379
 
-MAIL_DRIVER=smtp
-MAIL_HOST=smtp.mandrillapp.com
-MAIL_PORT=
-MAIL_USERNAME=
-MAIL_PASSWORD=
-
-S3_KEY=yourS3Key
-S3_SECRET=yourS3Secret
-S3_BUCKET=yourS3Bucket
-S3_URL=yourS3Url
-
-## SQS
-SQS_ACCESS_KEY_ID=
-SQS_SECRET_ACCESS_KEY=
-SQS_PREFIX=
-SQS_DEFAULT_QUEUE=
-
 NORTHSTAR_URL=https://identity-dev.dosomething.org
 NORTHSTAR_AUTH_ID=dev-oauth
 NORTHSTAR_AUTH_SECRET=
 NORTHSTAR_CLIENT_ID=dev-machine
 NORTHSTAR_CLIENT_SECRET=
 
-BLINK_URL=
-BLINK_USERNAME=
-BLINK_PASSWORD=
```

Comment:
Don't we need this for sending activity events to Customer.io?

Reply:
We do, but it's not something we want to be doing from local machines (since publishing an event to Blink might have side-effects, like triggering an email). Same argument as why we wouldn't want to encourage using S3 or SQS in the `.env.example`!

Reply:
Ahh, ok.